modules/nw-ovn-kubernetes-install-ovnkube-trace-local.adoc

```diff
@@ -19,14 +19,14 @@ The `ovnkube-trace` tool traces packet simulations for arbitrary UDP or TCP traf
 +
 [source,terminal]
 ----
-$  POD=$(oc get pods -n openshift-ovn-kubernetes -l app=ovnkube-master -o name | head -1 | awk -F '/' '{print $NF}')
+$  POD=$(oc get pods -n openshift-ovn-kubernetes -l app=ovnkube-control-plane -o name | head -1 | awk -F '/' '{print $NF}')
 ----
 
 . Run the following command on your local host to copy the binary from the `ovnkube-master` pods:
 +
 [source,terminal]
 ----
-$  oc cp -n openshift-ovn-kubernetes $POD:/usr/bin/ovnkube-trace ovnkube-trace
+$  oc cp -n openshift-ovn-kubernetes $POD:/usr/bin/ovnkube-trace -c ovnkube-cluster-manager ovnkube-trace
```

Comment:
ah the trace is now executed on each node so let' use ovnkube-node pod.
@pperiyasamy can you PTAL this part of the docs? -> I don't think we should be copying the binary to the cluster-manager? there is no database here.

Reply:
@tssurya This is about copying ovnkube-trace file from ovn pod into a remote node where it can be run as long as we've kubeconfig towards the cluster. so this change looks good to me.

Reply:
ah ack, its weird we pick the cluster-manager container because its not running on all nodes, i'd pick node one instead...